### PR TITLE
Add Simplified Setup for WSO2 EI 6.6.0

### PR DESCRIPTION
**Pull request by NishikaDeSilva**
## Purpose

> Add simplified setup for wso2 Enterprise Integrator 6.6.0

## Goal

>version upgrade

### CODE REVIEW

File: advanced/ei-pattern-1/auth.json [hunk 1]
```diff
@@ -7,4 +7,4 @@
 			"auth": "reg.auth"
 		}
 	}
-}
+}
```
Comment (chirangaalwis) [inline advanced/ei-pattern-1/auth.json]: @NishikaDeSilva add a new line EOF.

File: advanced/ei-pattern-2/auth.json [hunk 2]
```diff
@@ -7,4 +7,4 @@
 			"auth": "reg.auth"
 		}
 	}
-}
+}
```
Comment (chirangaalwis) [inline advanced/ei-pattern-2/auth.json]: @NishikaDeSilva add a new line EOF.

**Comment by chirangaalwis** [comment]
@NishikaDeSilva please create relevant issue(s) for these tasks.

File: advanced/ei-pattern-1/auth.json (hunk 1 above)
Comment (Aaquiff) [reply advanced/ei-pattern-1/auth.json]: @chirangaalwis we faced an issue after adding a newline for auth.json in APIM. Should we keep it that way?

**Pull request closed without merging**